server/methods/core/orders.js

```diff
@@ -64,6 +64,43 @@ export function ordersInventoryAdjust(orderId) {
   });
 }
 
+// TODO: Marketplace: Is there a reason to do this any other way? Can admins reduce for more
+// than one shop
+/**
+ * ordersInventoryAdjustByShop
+ * adjust inventory for a particular shop when an order is approved
```

Comment:
As part of removing the "approval" step, I think that inventory should be reduced when an order is placed (after copyCartToOrder) so that we avoid the possibility of multiple orders for the same inventory.

Reply:
I think removing Approval should probably be separate from this issue.